tests/providers/amazon/aws/hooks/test_sagemaker.py

```diff
@@ -528,7 +528,9 @@ def test_secondary_training_status_changed_false(self):
     def test_secondary_training_status_message_status_changed(self):
         now = datetime.now(tzlocal())
         SECONDARY_STATUS_DESCRIPTION_1["LastModifiedTime"] = now
-        expected_time = datetime.utcfromtimestamp(time.mktime(now.timetuple())).strftime("%Y-%m-%d %H:%M:%S")
+        expected_time = datetime.fromtimestamp(time.mktime(now.timetuple()), tz=timezone.utc).strftime(
+            "%Y-%m-%d %H:%M:%S"
+        )
```

Comment:
This can just be `datetime.now(tz=timezone.utc).strftime(...)` I think?

Reply:
Yeah I've also was confused by this construction when replace deprecated methods.

i think it could be replaced by
```now.astimezone(tz=timezone.utc).strftime("%Y-%m-%d %H:%M:%S")```